### PR TITLE
one GCP project share one default bucket

### PR DESCRIPTION
**Pull request by rmgogogo**
one project share one default bucket

### CODE REVIEW

File: manifests/gcp_marketplace/deployer/init_action.sh [hunk 1]
```diff
@@ -33,15 +32,13 @@ function set_bucket_and_configmap() {
...
+  gsutil ls ${bucket_name} || bucket_is_set=false
+  if [ "$bucket_is_set" = false ]; then
     bucket_is_set=true
-    bucket_name="hostedkfp-default-$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 10 | head -n 1)"
-    gsutil mb -p ${GCP_PROJECT_ID} "gs://${bucket_name}/" || bucket_is_set=false
-    if [ "$bucket_is_set" = true ]; then
-      break
-    fi
-  done
+    gsutil mb -p ${GCP_PROJECT_ID} "gs://${bucket_name}/" || bucket_is_set=false
```
Comment (Bobgy) [inline manifests/gcp_marketplace/deployer/init_action.sh:40]: Are there any concerns removing the retry?

Comment (rmgogogo) [reply manifests/gcp_marketplace/deployer/init_action.sh:40]: the retry is more for try different names

Comment (Bobgy) [reply manifests/gcp_marketplace/deployer/init_action.sh:40]: I see, SGTM

**Pull request merged**